### PR TITLE
[#158285444] Add calendar event for messages with due_date

### PR DESCRIPTION
**Pull request by fpersico**
For the calendar icon with the Month and Day on top i have used react-native-svg.

reminder

### CODE REVIEW

File: ios/ItaliaApp/Info.plist [hunk 1]
```diff
@@ -57,6 +57,10 @@
 	</dict>
 	<key>NSCameraUsageDescription</key>
 	<string>The app needs access to the camera to scan QR codes</string>
+  <key>NSCalendarsUsageDescription</key>
+	<string>The app needs access to the calendar to add event reminders</string>
+  <key>NSContactsUsageDescription</key>
+	<string>This app requires contacts access to function properly</string>
```
Comment (cloudify) [inline ios/ItaliaApp/Info.plist]: this may be a problem, is it really needed? @matteodesanti

File: ts/components/messages/MessageCTABar.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          <Button block={true} bordered={true} onPress={dispatchReminderAction}>
+            <Text>{I18n.t("messages.cta.reminder")}</Text>
+          </Button>
+        </View>
+      </View>
+    );
+  }
+
+  private renderSeparator() {
+    return <View style={styles.separatorContainer} />;
```
Comment (cloudify) [inline ts/components/messages/MessageCTABar.tsx]: why a method for a single component? it doesn't look like it saves much code :)

File: ios/ItaliaApp/Info.plist (hunk 1 above)
Comment (fpersico) [reply ios/ItaliaApp/Info.plist]: Look here https://github.com/vonovak/react-native-add-calendar-event/issues/3

**Comment by fpersico** [comment]
Refactored in https://github.com/teamdigitale/italia-app/pull/466

**Pull request closed without merging**